htdocs/api/v0.0.3-dev/APIBase.php

```diff
@@ -79,6 +79,14 @@ function __construct($method)
                 $this->error("User not authenticated");
                 $this->safeExit(0);
             }
+
+            $user = \User::singleton();
+            // If the user has the medhub permission they will be blocked from any endpoints not containing medhub
+            if (!str_contains($_SERVER['REQUEST_URI'],"/medhubConsent") && $user->hasPermission('medhub')) {
```

Comment:
@KLaFleur notice the permissions that was required here and loo kat your new code. that permission is gone